types/clownface/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+      "rdf-js": "^2.0"
```

Comment:
Reading https://github.com/rdfjs/types/pull/4#issuecomment-833273515 looks like this will exist as a proxy for `@rdfjs/types`.

Reply:
Yes, that is in order not to break existing imports from `rdf-js`. New could would be encouraged to import from the `@rdfjs/types` package